ext/redhat/puppet.spec.erb

```diff
@@ -336,8 +338,8 @@ fi
 if [ "$1" -ge 1 ]; then
   # The pidfile changed from 0.25.x to 2.6.x, handle upgrades without leaving
   # the old process running.
-  oldpid="%{_localstatedir}/run/puppet/puppetmasterd.pid"
-  newpid="%{_localstatedir}/run/puppet/master.pid"
+  oldpid="%{_localstatedir}/run/puppetlabs/puppetmasterd.pid"
```

Comment:
The oldpid is probably in the same old place?

And do we even need this bit anymore? /cc @haus